vercel.json

```diff
@@ -0,0 +1,9 @@
+{
+  "buildCommand": "pnpm install && mkdir -p dist && for pkg in */src; do if [ -f \"$pkg/package.json\" ]; then pkg_name=$(dirname \"$pkg\") && cd \"$pkg\" && pnpm build --base /$pkg_name --out ../../dist/$pkg_name && cd ../..; fi done",
+  "outputDirectory": "dist",
+  "installCommand": "pnpm install",
+  "framework": null,
+  "buildEnvironment": {
```

Comment:
The `BASE_PATH` environment variable was only for GitHub Actions build.
If this doesn't have any effect on Vercel, delete it.